pkg/proxy/proxy.go

```diff
@@ -651,3 +507,20 @@ func (p *Proxy) GetStatusModel() *models.ProxyStatus {
 		Redirects: p.getRedirectsStatusModel(),
 	}
 }
+
+// LocalEndpointInfo return an EndpointInfo with the information of the local endpoint
+func (r *Redirect) LocalEndpointInfo() accesslog.EndpointInfo {
```

Comment:
Remove. This doesn't seem to be called anywhere anymore.

Reply:
```
pkg/proxy/logger/logger.go-     return func(lr *LogRecord) {
pkg/proxy/logger/logger.go-             switch lr.ObservationPoint {
pkg/proxy/logger/logger.go-             case accesslog.Ingress:
pkg/proxy/logger/logger.go:                     lr.DestinationEndpoint = lr.producer.LocalEndpointInfo()
pkg/proxy/logger/logger.go-             case accesslog.Egress:
pkg/proxy/logger/logger.go:                     lr.SourceEndpoint = lr.producer.LocalEndpointInfo()
pkg/proxy/logger/logger.go-             }
pkg/proxy/logger/logger.go-
pkg/proxy/logger/logger.go-             ipstr, port, err := net.SplitHostPort(i.SrcIPPort)
```